test/e2e/port_forward.go

```diff
@@ -7,19 +7,22 @@ import (
 	"net/url"
 	"strings"
 
+	"github.com/stretchr/testify/require"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/portforward"
 	"k8s.io/client-go/transport/spdy"
 )
 
 // CreatePortForward listens for local connections and forwards them to a remote pod
-func CreatePortForward(namespace, pod string, ports []string, kConfig *rest.Config) (*portforward.PortForwarder, chan struct{}, error) {
+func CreatePortForward(namespace, podNamePrefix, containsImage string, ports []string, kConfig *rest.Config) (*portforward.PortForwarder, chan struct{}) {
 	roundTripper, upgrader, err := spdy.RoundTripperFor(kConfig)
 	if err != nil {
-		return nil, nil, err
+		printTestStackTrace()
+		require.NoError(t, err)
 	}
 
-	path := fmt.Sprintf("/api/v1/namespaces/%s/pods/%s/portforward", namespace, pod)
+	pod := GetPod(namespace, podNamePrefix, containsImage, fw.KubeClient)
```

Comment:
Isn't the error check missing here?

Reply:
No, because I also updated GetPod so it will fail if it does not find the pod.

Reply:
Sorry missed that.